examples/imagenet/README.md

```diff
@@ -66,6 +66,11 @@ python main.py \
 #### Running on Cloud
 
 See commands in [../cloud/README.md](../cloud/README.md)
+
+To use a dataset stored in a bucket, set the following shell parameter:
+
+    export TFDS_DATA_DIR="gs://$GCS_TFDS_BUCKET/tensorflow_datasets"
```

Comment:
This is described in more detail under `## Training the imagenet example` in ../cloud/README.md - why do you think it is helpful to add this comment?

Maybe it would be better to note that there is a longer description about how to prepare the `imagenet2012` dataset in that other README.

Reply:
As far as I can tell, it's not documented in the cloud/README.md file, am I missing something?  The only reference I see is https://github.com/google/flax/blob/4fd597c1d4c987d561abdb19f34eca58f15c7743/examples/cloud/startup_script.sh#L58 where the python script is setting said shell variable for the user.  My reference point into this is running flax on a TPU directly and so by extension having to manually set these variables to get the demo to work.

Reply:
Got it, LGTM in that case.